scripts/find_untracked_release_tests.py

```diff
@@ -0,0 +1,88 @@
+import os
+import yaml
+import tempfile
+import re
+import pprint
+
+pp = pprint.PrettyPrinter(indent=4)
+tempdir = tempfile.mkdtemp()
+
+def ci_covered_release_tests(root_dir, pre_fix="repos/"):
+    covered_release_test_names = set()
+    for path, _, files in os.walk(root_dir):
+        for filename in files:
+            filepath = os.path.join(path, filename)
+            with open(filepath, "r") as f:
+                content = yaml.safe_load(f)
+
+            for item in content:
+                if "path" in item.keys():
+                    test_path = item["path"]
+                    
+                    # Remove pre-fix
+                    if test_path.startswith(pre_fix):
+                        test_path = test_path[len(pre_fix):]
+                        covered_release_test_names.add(test_path)
+
+    return covered_release_test_names
+
+def find_existing_examples(repo_name, github_addr, examples_dir, require_entrance=True):
+    repo_dir = f"{tempdir}/{repo_name}/"
+    os.system(f"git clone {github_addr} {repo_dir}")
```

Comment:
This can fail, and please add `--max-depth=1` to avoid pulling full history (to save time).